spec/amp-html-format.md

```diff
@@ -126,11 +126,11 @@ HTML tags can be used unchanged in AMP HTML. Certain tags have equivalent custom
 | param     | Prohibited. |
 | applet    | Prohibited. |
 | embed     | Prohibited. |
-| form      | Prohibited. |
+| form      | Prohibited. [Support coming in the future.](https://github.com/ampproject/amphtml/issues/1286) |
 | input elements | Prohibited. Includes input, textarea, select, option. Notably, button element is allowed. |
 | button    | Allowed. |
 | <a name="cust"></a>style     | [Required style tags for adjusting opacity](#opacity) One additional style tag is allowed in head tag for the purpose of custom styling. This style tag must have the attribute `amp-custom`. [🔗](#cust) |
-| link      | Allowed for certain values of rel: `canonical`. `stylesheet` is generally disallowed, but some values may be whitelisted for font providers. |
+| link      | `rel` values registered on [microformats.org](http://microformats.org/wiki/existing-rel-values) are allowed. If a rel value is missing from our whitelist, [please submit an issue](https://github.com/ampproject/amphtml/issues/new). `stylesheet` is generally disallowed, but some values may be whitelisted for font providers. |
```

Comment:
Technically this isn't enough detail. Some of those rel values are intentionally not validating. Essentially anything that has a behavior in a major browser which would cause network activity or change rendering, I dropped from the validation list. For a full list, I think these cover all of the exceptions: rel=import, rel=components, rel=prerender, rel=subresource, rel=preconnect, rel=prefetch, rel=preload.